### PR TITLE
Update PR template to be more explicit about expectations to check off everything

### PR DESCRIPTION
**Pull request by flodnv**
### Details

I have seen some confusions, hopefully this clears them up.
cc @Expensify/contributor-plus 

### Fixed Issues
https://expensify.slack.com/archives/C01SKUP7QR0/p1655892677571859

### Tests

none needed

### PR Review Checklist

this is what I am modifying 

### QA Steps

N/A

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
...
     - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
-- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
+- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers. Check this off if it does not apply.
 - [ ] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
-- [ ] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
+- [ ] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected). Check this off if it does not apply.
```
Comment (flodnv) [inline .github/PULL_REQUEST_TEMPLATE.md:64]: Is something related to this line missing from the C+ checklist?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -59,12 +59,12 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
...
 - [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
 - [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
-- [ ] If a new component is created I verified that:
+- [ ] If a new component is created I verified that (check these off if they do not apply):
```
Comment (sketchydroide) [inline .github/PULL_REQUEST_TEMPLATE.md:67]: hmm, seems you have two different patterns for the same thing:
1. `. Check this off if it does not apply.`
2. `(check these off if they do not apply)` 

Not sure why we can't use `(check these off if they do not apply)` everywhere
And on the items that have subitems, it's not clear to me if we should only check the parent item, or the parent and all of the children.

**Review by sketchydroide — CHANGES_REQUESTED** [review]
just a small question
Happy to aprove otherwise

**Review by iwiznia — COMMENTED** [review]
Should we instead add this as instructions in the top? ie: instead of adding `check this off if it does not apply` to every item, let's just say so on the top

**Comment by sketchydroide** [comment]
Not sure, I mean, no body reads is a thing, and this makes sure it's read, I think I prefer to have it in every line

**Comment by parasharrajat** [comment]
I agree with @iwiznia. Why not add a subtitle to the `PR Review Checklist` title that says `Check off the items that do not apply.`

**Comment by mananjadhav** [comment]
With some of my projects where we don't have such a large checklist, we use the pattern of marking the item checked and striking through to confirm it doesn't apply. Example

[x] ~ Fork the Postman Collection if you have made an edit in the API contract~

**Comment by puneetlath** [comment]
I also agree that adding this at the top would be less clutter than having it in each line. Especially because we'll add new lines and will have to remember to add this to those. 

Alternatively, we could add a last checkbox of something like `I have checked off every box in the PR author checklist, including those that don't apply to this PR)`

**Comment by sketchydroide** [comment]
Oh I like that @puneetlath

**Comment by mallenexpensify** [comment]
> With some of my projects where we don't have such a large checklist, we use the pattern of marking the item checked and striking through to confirm it doesn't apply. 

I really like @mananjadhav's strikethrough idea because it helps differentiate items that were checked vs those that aren't applicable (where the box is still checked). Can anyone think of any downside to this approach?

**Comment by sketchydroide** [comment]
>I really like @mananjadhav's strikethrough idea because it helps differentiate items that were checked vs those that aren't applicable (where the box is still checked). Can anyone think of any downside to this approach?

it doesn't sound like a bad pattern, but I don't know how to apply the strikethrough without editing the text, and that can be a bit annoying to do in such a large list of things to check (and maybe leading to do it on the wrong line).

**Comment by flodnv** [comment]
I agree! Quite obvious in hindsight indeed! I redid the PR: https://github.com/Expensify/App/pull/9698

**Pull request closed without merging**